packages/vscode-container-client/src/test/PodmanClient.test.ts

```diff
@@ -0,0 +1,391 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as stream from 'stream';
+import { describe, it } from 'mocha';
+import { PodmanClient } from '../clients/PodmanClient/PodmanClient';
+// import { ShellStreamCommandRunnerFactory } from '../commandRunners/shellStream';
+import { WslShellCommandRunnerFactory } from '../commandRunners/wslStream';
+import { expect } from 'chai';
+import { PodmanListImageRecord } from '../clients/PodmanClient/PodmanListImageRecord';
+
+const testDockerUsername = '';
+const testDockerPat = '';
+
+const client = new PodmanClient();
+const runner = new WslShellCommandRunnerFactory({ strict: true });
+const testDockerfileContext = '/mnt/d/vscode-docker-extensibility/packages/vscode-container-client/src/test/buildContext';
+const testDockerfile = '/mnt/d/vscode-docker-extensibility/packages/vscode-container-client/src/test/buildContext/Dockerfile';
+
+// const client = new PodmanClient();
+// const runner = new ShellStreamCommandRunnerFactory({ strict: true });
+// const testDockerfileContext = 'D:\\vscode-docker-extensibility\\packages\\vscode-container-client\\src\\test\\buildContext';
+// const testDockerfile = 'D:\\vscode-docker-extensibility\\packages\\vscode-container-client\\src\\test\\buildContext\\Dockerfile';

```

Comment:
Remove commented out code

Reply:
The tests are meant to be run locally, with some things commented in/out, so I'd prefer to keep this code. I agree though that the paths should be changed to be relative.